zql/docs/processors/README.md

```diff
@@ -231,6 +232,38 @@ ID.ORIG_H     ID.ORIG_P ID.RESP_H       ID.RESP_P ORIG_BYTES RESP_BYTES TOTAL_BY
 10.47.5.155   40728     91.189.91.23    80        21396      98972170   98993566
 ```
 
+---
+
+## `rename`
+
+|                           |                                                 |
+| ------------------------- | ----------------------------------------------- |
+| **Description**           | Rename fields in a record. |
+| **Syntax**                | `rename <newname> = <oldname> [, <newname> = <oldname> ...]`     |
+| **Required arguments**    | One or more field assignment expressions. Renames are applied left to right; each rename observes the effect of all renames that preceded it. |
+| **Optional arguments**    | None |
+| **Limitations**           | A field can only be renamed within its own record. For example id.orig_h can be renamed to id.src, but it cannot be renamed to src. |
```

Comment:
```suggestion
| **Limitations**           | A field can only be renamed within its own record. For example `id.orig_h` can be renamed to `id.src`, but it cannot be renamed to `src`. |
```

I've been trying to stick to a convention in the docs of using fixed width when referencing field names.